nbconvert/templates/html/slides_reveal.tpl

```diff
@@ -211,13 +209,29 @@ require(
 
         Reveal.addEventListener('slidechanged', update);
 
-        var update_scroll = function(event){
-          $(".reveal").scrollTop(0);
-        };
-
-        Reveal.addEventListener('slidechanged', update_scroll);
+        function setScrollingSlide(scroll) {
+
+            if (scroll === "True") {
+              var h = $('.reveal').height() * 0.95;
+              var hpx = "" + h + "px";
+              $('section.present').find('section')
+                .filter(function() {
+                  return $(this).height() > h;
+                })
+                .css('height', hpx)
```

Comment:
This looks along the lines of the kinds of things to get around `'auto'` heights in the case of animations. There should be a way to work natively in pixels rather than treating it as a string manipulation problem though. I forget what that is off the top of my head though.